src/Hash.hpp

```diff
@@ -170,6 +171,13 @@ inline void HashAddString(HashState *self, const char *s)
     HashUpdate(self, s, strlen(s));
 }
 
+inline void HashAddString(FILE* debug_hash_fd, HashState* state, const char* label, const char* str)
```

Comment:
We have the logging `HashAddString` in the header and `HashAddInteger` in the cpp. Let's put them into consistent places.

Reply:
This because the implementation of HashAddString is in the header, as it's only a small wrapper around HashAddUpdate